config.tfvars

```diff
@@ -42,6 +42,9 @@ desired_capacity = 1
 
 # Helm chart version of Jira
 jira_helm_chart_version = "1.2.0"
+# The version of Jira software is defaulted in Helm chart. If you wish to install a different
+# version of Jira software you can uncomment the following line and define your desired version.
+#jira_software_version   = "8.20.5-jdk11"
```

Comment:
I am not sure if we should include default, even commented out. This means we need to keep it up to date as customers might uncomment the line thinking that this is the default (from helm chart). I think we can leave this empty. As we point them to the docker hub tags (comment above), they can see examples of the versions that are available.